drivers/usbhid-ups.c

```diff
@@ -2086,6 +2093,9 @@ static void ups_alarm_set(void)
 	if (ups_status & STATUS(BYPASSMAN)) {
 		alarm_set("Manual bypass mode!");
 	}
+	if (ups_status & STATUS(ECOMODE)) {
+		alarm_set("ECO(HE) mode!");
+	}
```

Comment:
I think this is something we should drop since we're handling `ECO` mode as a status? Reason being users might choose to run their UPS in ECO mode more frequently and this would raise unnecessary concerns (alarms, ha 😆 ) and/or make them "blind" to actual alarms worth being concerned about as a side-effect. 😉

Reply:
Makes sense, eventually to revise in the other driver(s?) that raise an ECO alarm...